etc/inc/disable-programs.inc

```diff
@@ -19,7 +19,6 @@ blacklist ${HOME}/.Sayonara
 blacklist ${HOME}/.Steam
 blacklist ${HOME}/.Steampath
 blacklist ${HOME}/.Steampid
-blacklist ${HOME}/.TelegramDesktop
```

Comment:
This directory might still exist on some systems and it maybe contains things like chat protocols? We should keep the `blacklist` of it.